app/assets/javascripts/react_components/admin_dashboard/admin_dashboard.js.jsx

```diff
@@ -162,6 +162,9 @@ var Partner = React.createClass({
   getInitialState: function () {
     return {clicked: false};
   },
+  componentDidMount: function() {
+    this.setState({clicked: this.props.selectedPartner == this.props.partnerId});
```

Comment:
Ok I understand what you were talking about now haha.

But there is a much cleaner way of writing this, so try to do it like this:

This will be a lot simpler if we just don't have the state variable in the first place. So you can remove the `getInitialState`, `componentDidMount` and `componentWillReceiveProps` methods entirely and then have:

```
var clicked = this.props.selectedPartner == this.props.partnerId;
var is_active = clicked ? "active" : "";
```

I'm pretty sure this will do everything you want it to do.